perun-core/src/main/java/cz/metacentrum/perun/core/implApi/GroupsManagerImplApi.java

```diff
@@ -570,13 +559,22 @@
 	boolean isOneWayRelationBetweenGroups(Group resultGroup, Group operandGroup) throws InternalErrorException;
 
 	/**
-	 * Return all group relations.
+	 * Return all result groups of requested operand group.
 	 *
 	 * @param sess perun session
 	 * @param groupId group id
 	 * @return list of group ids
 	 * 
 	 * @throws cz.metacentrum.perun.core.api.exceptions.InternalErrorException
 	 */
-	List<Integer> getGroupRelations(PerunSession sess, int groupId) throws InternalErrorException;
+	List<Integer> getResultGroups(PerunSession sess, int groupId) throws InternalErrorException;
```

Comment:
Honestly I'm confused about the original method names. We had ``getGroupRelations()`` and ``getRelatedGroupsIds()`` which both returned list of result groups ids by operand gid.
I believe first should return Groups and second Ints. But they both returned ints. This was probably wrong in original implementation and is reason we didn't have getter method on relations.

So, right now it seems to be ok to use ``getResultGroups()`` and ``getOperandGroups()``but I was hoping for a single joining select for groups, instead of using ``getGroupsByIds()`` in BL layer. Can you please bring back method ``getResultGroupsIds()`` (changed name!) which can be used for simple checks in member handling methods ? But please without ``operand_gid=" + groupId`` like constructs in calling db from java (sql injection). And than convert new methods to return List<Group> from Impl. Thank you.